pkg/endpoint/endpoint_status.go

```diff
@@ -325,7 +325,6 @@ func (e *Endpoint) GetCiliumEndpointStatus(conf EndpointStatusConfiguration) *ci
 		ExternalIdentifiers: modelStatus.ExternalIdentifiers,
 		Identity:            getEndpointIdentity(modelStatus),
 		Networking:          getEndpointNetworking(modelStatus),
-		State:               compressEndpointState(modelStatus.State),
```

Comment:
This was not the intent. The intent was to publish a state that doesn't change consistently. Why are you making this change? Did you observe many updates? Or are you observing another PATCH required after the initial CREAT?

Reply:
Reverted